.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -653,6 +653,12 @@ chmod 700       "${SUBMITTY_INSTALL_DIR}/.setup/bin/reupload"*
 chown root:root "${SUBMITTY_INSTALL_DIR}/.setup/bin/track_git_version.py"
 chmod 700       "${SUBMITTY_INSTALL_DIR}/.setup/bin/track_git_version.py"
 
+###############################################
+# submitty_test script
+cp  "${SUBMITTY_REPOSITORY}/.setup/SUBMITTY_TEST.sh"        "${SUBMITTY_INSTALL_DIR}/.setup/"
+chown root:root "${SUBMITTY_INSTALL_DIR}/.setup/SUBMITTY_TEST.sh"
+chmod 700       "${SUBMITTY_INSTALL_DIR}/.setup/SUBMITTY_TEST.sh"
+
```

Comment:
Is this necessary?  I'm not familiar enough with the setup scripts and permissions structure to comment with any certainty, but I think it would be good for @bmcutler to take a look before this gets merged.

Reply:
I would say it is a good practice, to make sure script has executable bit. Without chmod, it won't. 
Although github now tracks the executable bit, it doesn't work for our case.

Reply:
Yes, but should we be installing this script on all systems?

Reply:
I guess we should have it installed, and it is up to devs to use it or not.

Reply:
I guess the concern is in the production server right?

Reply:
Yep.